### PR TITLE
AES-CMAC 128 Message [CLDAPPS-1503]

### PR DESCRIPTION
**Pull request by RReichert**
# Description

Adding a new "AES CMAC signature" message.

image

# API compatibility

No, this is a new message.

## API compatibility plan

N/A

# JIRA Reference

https://swift-nav.atlassian.net/browse/CLDAPPS-1503

### CODE REVIEW

File: spec/yaml/swiftnav/sbp/signing.yaml [hunk 1]
```diff
@@ -177,6 +177,56 @@ definitions:
...
+        - certificate_id:
+            type: array
+            fill: u8
+            size: 4
+            desc: The last 4 bytes of the certificate's SHA-1 fingerprint
+        - signature:
+            type: array
+            fill: u8
+            size: 16
+            desc: Signature
```
Comment (ljbade) [inline spec/yaml/swiftnav/sbp/signing.yaml:209]: a bit nit picky but the correct term is a tag for a MAC, maybe `Signature (CMAC tag value)` will satisfy

**Review by ljbade — APPROVED** [review]

**Review by notoriaga — APPROVED** [review]

**Review by notoriaga — APPROVED** [review]
I can take a look at the rust stuff tomorrow

**Review by sokhealy — APPROVED** [review]
looks good!

**Review by pcrumley — APPROVED** [review]
:shipit:

**Pull request merged**